### PR TITLE
tools: add Reparo User Guide

### PR DESCRIPTION
**Pull request by CaitinChen**
Via https://github.com/pingcap/docs-cn/pull/973
@WangXiangUSTC @lilin90 PTAL

### CODE REVIEW

File: tools/reparo.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+## Reparo usage
+
+### Description of command line parameters
+
+```
+Usage of Reparo:
+-L string
+    The level of the output information of logs
+    Value: "debug"/"info"/"warn"/"error"/"fatal" ("info" by default)
+-V The print version
```
Comment (WangXiangUSTC) [inline tools/reparo.md]: print the version?

File: tools/reparo.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+# Value: "debug"/"info"/"warn"/"error"/"fatal" ("info" by default)
+log-level = "info"
+
+# Uses `start-datetime` and `stop-datetime` to specify the time range in which 
+# the binlog files are to be recovered.
+# Format: "2006-01-02 15:04:05"
+# start-datetime = ""
+# stop-datetime = ""
+
+# Correspond to `tso` of `start-datetime` and `stop-datetime` respectively.
```
Comment (lilin90) [inline tools/reparo.md]:
```suggestion
# Correspond to `start-datetime` and `stop-datetime` respectively. 
```

File: tools/reparo.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+```
+
+### Note
+
+* `data-dir` specifies the directory for the binlog file which Drainer outputs.
+* Both `start-datatime` and `start-tso` are used to specify the time point for starting recovery, but they are different in the time format. If they are not set, the recovery process starts from the earliest binlog file by default.
+* Both `stop-datetime` and `stop-tso` are used to specify the time point for finishing recovery, but they are different in the time format. If they are not set, the recovery process ends up with the last binlog file by default.
+* `dest-type` specifies the destination type. Its value can be "mysql" and "print." 
+
+    * When it is set to｀mysql｀, the data can be recovered to MySQL or TiDB that uses or is compatible with the MySQL protocol. In this case, you need to specify the database information in [dest-db] of the configuration information.
```
Comment (lilin90) [inline tools/reparo.md]:
```suggestion
    * When it is set to｀mysql｀, the data can be recovered to MySQL or TiDB that uses or is compatible with the MySQL protocol. In this case, you need to specify the database information in `[dest-db]` of the configuration information.
```

File: tools/reparo.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    * When it is set to ｀print｀, only the binlog information is printed. It is generally used for debugging and checking the binlog information. In this case, there is no need to specify [dest-db].
```
Comment (lilin90) [inline tools/reparo.md]:
```suggestion
    * When it is set to ｀print｀, only the binlog information is printed. It is generally used for debugging and checking the binlog information. In this case, there is no need to specify `[dest-db]`.
```

**Comment by CaitinChen** [comment]
@lilin90 Thanks. PTAL again.

**Comment by CaitinChen** [comment]
@lilin90 PTAL again~

**Pull request merged**